### PR TITLE
chore(support): Improve bug/feedback/support modal experience

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

A few improvements to the reporting experience, mostly coming from experience handling some Product Analytics messages today:
- the modal title and field labels are now specific to the selected message type
- reworded "What kind of request is this?" to "What type of message is this?" as "What kind of ... is this" sounds a bit aggressive, and it's more of a message than a request
- removed the "won't get back to everyone individually" bit, as it was very wordy and in some dismissive – I think we can convey this more clearly by wording this is a "message" rather than a "request"
- added image upload to the message
- fixed an issue where `LemonSelect` wouldn't immediately scroll down to the selected option in a long menu (such as the support area one) - you no longer have to scroll manually now
- refactored `HelpButton` to a `LemonMenu`
- added "support" as a third message type (e.g. today I answered to a "please change organization owner" request, for which there was _no_ correct classification with just "bug" and "feedback")
- divided `HelpButton` into sections for visual clarity, and put the choice of bug/feedback/support right in that menu, so that users know what are the use cases at first glance
- fixed alignment of modal footer buttons (Cancel/Submit)

| | Before | After |
|--------|--------|--------|
| Button | before-but | after-but |
| Modal | before-mod | after-mod |

### CODE REVIEW

File: frontend/src/lib/components/Support/supportLogic.ts [hunk 1]
```diff
@@ -28,24 +29,24 @@ function getDjangoAdminLink(user: UserType | null): string {
...
-    data_management: 'Data Management',
     data_integrity: 'Data Integrity',
-    ingestion: 'Events Ingestion',
+    data_management: 'Data Management',
+    ingestion: 'Event Ingestion',
     experiments: 'Experiments',
     feature_flags: 'Feature Flags',
-    login: 'Login / Sign up / Invites',
-    session_replay: 'Session Replay',
+    analytics: 'Product Analytics (Insights)',
```
Comment (tiina303) [inline frontend/src/lib/components/Support/supportLogic.ts:43]: Maybe:
```suggestion
    analytics: 'Product Analytics (Insights, dashboards, event explorer, annotations)',
```
You can also add them separately if you prefer making sure to update the url path mappings too. These were previously was covered under "Analytics" name kindof...

**Review by tiina303 — APPROVED** [review]
qq: Does the modal update if I change from bug to feedback - would be odd to see the bug prompt text after I switched to feedback. We could hide it completely by default, but note that there are places in the app, where I'd want to click into it and have the user select what it was later 🤷‍♀️  or that's how it currently works

File: frontend/src/lib/components/Support/supportLogic.ts (hunk 1 above)
Comment (Twixes) [reply frontend/src/lib/components/Support/supportLogic.ts:43]: Mostly added this just to disambiguate, as I think for some users we are a product analytics platform, so I've seen them select this even if their particular request is not really relevant to the product analytics area.

**Comment by Twixes** [comment]
Yup, the modal title is updated when the users switches. Gonna keep the message type selector for exactly the reason you mentioned.

**Pull request merged**